app/views/help/ab_testing.html.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :title, "A/B testing - GOV.UK" %>
+<% content_for :extra_headers do %>
+  <meta name="description"
+  <% # FIXME: Does this page need specific wording? %>
```

Comment:
Can we add a `noindex` tag so that Google won't pick this up? Perhaps the description can be left off then.